File: Solution in C++/Assignment_02/Task12.cpp
#include<iostream>

using namespace std;

int main(){
    int x;
    cin>>x;
    if(x<0 || x>24){cout<<"Wrong Time";}
    else if(4<=x<=6){cout<<"Breakfast";}
    else if(12<=x<=13){cout<<"Lunch";}
    else if(16<=x<=17){cout<<"Snacks";}
    else if(19<=x<=20){cout<<"Dinner";}
    else{cout<<"Patience is a virtue";}
}

File: Solution by Python/Assignment_02/Task12.py
n=int(input())
if n>=4 and n<=6:
    print("Breakfast")
elif n>=12 and n<=13:
    print("Lunch")
elif n>=16 and n<=17:
    print("Snacks")
elif n>=19 and n<=20:
    print("Dinner")
elif n>=23 or n<=0:
    print("Wrong time")
else:
    print("Patience is a virtue")